flipperdevices/flipperzero-firmware
# [FL-2344] iButton deleted back button

# What's new

- Info scene: removed "Back" button and changed layout
- Delete confirm scene: Renamed "Back" into "Cancel"

# Verification 

- Open ibutton key info and check that there is no left "Back" button anymore
- Try to delete any key and check that left button named "Cancel"

# Checklist (For Reviewer)
